app/src/repositories/CardsRepository.py

```diff
@@ -8,23 +9,22 @@
 
 conn = dbConn.get()
 
-#TODO implement exception handling 
+#TODO implement exception handling
 def create(deck_id: int, word: str, translation: str) -> Optional[Card]:
 
     query = f"INSERT INTO card (deck_id, word, translation) VALUES ({deck_id}, '{word}', '{translation}');"
 
     c = conn.cursor()
 
     c.execute(query)
-
     conn.commit()
 
     card_id = c.lastrowid
 
     if card_id:
         logger.debug(
             f"Card with id {card_id} has been saved into database successfully")
-        return Card(card_id, deck_id, word, translation)
+        return get_by_id(card_id)
```

Comment:
why that change? now you do additional sql query

Reply:
done